libs/libdvbcsa/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2020 Rafał Dzięgiel <rafostar.github@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdvbcsa
+PKG_RELEASE:=1
+
+PKG_SOURCE_VERSION:=bc6c0b164a87ce05e9925785cc6fb3f54c02b026
+PKG_HASH:=2d761c9e094642f2c9aa7e66534c6147a59d0d0bc709ec0f2fdbb34bf020d8ec
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.gz
+PKG_SOURCE_URL:=https://code.videolan.org/videolan/libdvbcsa/-/archive/$(PKG_SOURCE_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_SOURCE_VERSION)
+
+PKG_MAINTAINER:=Rafał Dzięgiel <rafostar.github@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libdvbcsa
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=DVB Common Scrambling Algorithm Library
+  URL:=https://www.videolan.org/developers/libdvbcsa.html
+endef
+
+define Package/libdvbcsa/description
+ Libdvbcsa is a free implementation of the DVB Common Scrambling
+ Algorithm DVB/CSA - with encryption and decryption capabilities.
+endef
+
+define Package/libdvbcsa/config
+  menu "Configuration"
+  depends on PACKAGE_libdvbcsa
+  source "$(SOURCE)/Config.in"
+  endmenu
+endef
+
+ifeq ($(CONFIG_LIBDVBCSA_DEBUG),y)
+  CONFIGURE_ARGS += --enable-debug
+endif
+
+ifeq ($(CONFIG_LIBDVBCSA_MMX),y)
+  CONFIGURE_ARGS += --enable-mmx
+else ifeq ($(CONFIG_LIBDVBCSA_SSE2),y)
+  CONFIGURE_ARGS += --enable-sse2
+else ifeq ($(CONFIG_LIBDVBCSA_ALTIVEC),y)
+  CONFIGURE_ARGS += --enable-altivec
+else ifeq ($(CONFIG_LIBDVBCSA_NEON),y)
+  CONFIGURE_ARGS += --enable-neon
+endif
+
+define Build/InstallDev
```

Comment:
This is a bit broad. Please be more verbose in this section.

Reply:
Only one can be used at a time. In `configure.ac` the used CPU extension is selected inside long `if... else if...` statement. Selecting more than one would not break the compiling, but we would end up with library optimized for the CPU on top of the `if-else` statement.

This too can be simplified with `--$(if $(CONFIG_LIBDVBCSA_SSE2),en,dis)able-sse2` like you pointed above, because I made the `Config.in` in a way that prevents selecting more then one option per arch.

Reply:
Not what I meant. The InstallDev section is not verbose as it is in other packages.

Reply:
@neheb I updated this part in PR. Is this what you wanted?

Reply:
Yes